core/classes/Database/PhinxAdapter.php

```diff
@@ -73,4 +121,52 @@ public static function migrate(): string {
         return $output;
     }
 
+    /**
+     * Rolls back migrations
+     * Logs output of Phinx to other-log.log file
+     *
+     * @param string $module Module name
+     * @param string $migrationDir Migration directory to use
+     * @param int $since Version of earliest migration to rollback, default 0 for all
+     *
+     * @return string Output of the migration command from Phinx as if it was executed in the console.
+     *
+     * @throws Exception If unable to rollback
+     */
+    public static function rollback(
```

Comment:
Good addition - I wonder where/how we could implement this

Reply:
The use case I had in mind was for `onUninstall` for modules - we can remove the database tables when uninstalling a module